src/structures/GuildMember.js

```diff
@@ -62,7 +62,7 @@ class GuildMember extends Base {
        * The user that this guild member instance represents
        * @type {?User}
        */
-      this.user = this.client.users._add(data.user, true);
+      this.user = this.client.users._add(data.user, true)._clone();
```

Comment:
@vladfrangu I added a setting for it, hopefully you could rereview your standpoint as it's better as a setting since most peeps indeed don't need this.